src/components/modals/RegistrationModal.js

```diff
@@ -0,0 +1,147 @@
+import React, { useState } from 'react'
+import styles from './Modal.module.css'
+import PropTypes from 'prop-types'
+import { ReactComponent as ExitLogo } from '../icons/exit.svg'
+import { ReactComponent as Eye } from '../icons/eye.svg'
+import { ReactComponent as EyeNo } from '../icons/eye_no.svg'
+
+const RegistrationModal = ({ setIsRegistrationModalVisible, setToken }) => {
+    const [isPaswordShow, setIsPaswordShow] = useState(false)
+    const [isPasword2Show, setIsPasword2Show] = useState(false)
+    const [isPassSame, setIsPassSame] = useState(true)
+    const [registrationError, setRegistrationError] = useState('')
+    const [isLoading, setIsLoading] = useState(false)
+    const [email, setEmail] = useState('')
+    const [pass1, setPass1] = useState('')
+    const [pass2, setPass2] = useState('')
+
+    const formData = {
+        email,
+        username: email,
+        password: pass1,
+    }
+
+    const passAlert = () => {
+        if (pass1 !== pass2) {
+            setIsPassSame(false)
+        }
+    }
+
+    const registration = e => {
+        e.preventDefault()
+        fetch(
+            'https://radiant-temple-07706.herokuapp.com/auth/local/register',
+            {
+                method: 'POST',
+
+                headers: {
+                    'Content-Type': 'application/json',
+                },
+                body: JSON.stringify(formData),
+            }
+        )
+            .then(data => data.json())
+            .then(body => {
+                if (body.jwt) {
+                    setToken(body.jwt)
+                    setIsLoading(false)
+                    setIsRegistrationModalVisible(false)
+                } else {
+                    setIsLoading(false)
+                    setRegistrationError('Email уже используется')
+                }
```

Comment:
since you use `setIsLoading(false)` in both cases, you can just put them right after both conditions